### ISSUE TITLE
graal/vm suite: mx_vm.py is unable to match JDK string of AdoptOpenJDK

### ISSUE THREAD

**Issue by oroppas** [oracle/graal#1304]
```
env MX_HOME='C:\msys64\home\cd4mcq\packages\mx' HOME='C:\Users\cd4mcq' MX__SUITEMODEL=sibling GRAAL_PYTHONHOME='C:\msys64\home\cd4mcq\packages\graalpython\graalpython' MX_SUBPROCESS_COMMAND_FILE='c:\users\cd4mcq\appdata\local\temp\mx_subprocess_command.w8_ykv' \
'C:\Program Files\AdoptOpenJDK\jdk-11.0.3.7-hotspot\bin\java.exe' -cp 'C:\msys64\home\cd4mcq\packages\graal\compiler\mxbuild\.jdk11.0.3' org.graalvm.compiler.hotspot.JVMCIVersionCheck
  File "C:\msys64\home\cd4mcq\packages\mx\mx.py", line 19191, in <module>
    main()
  File "C:\msys64\home\cd4mcq\packages\mx\mx.py", line 19102, in main
    primary_suite().recursive_post_init()
  File "C:\msys64\home\cd4mcq\packages\mx\mx.py", line 9134, in recursive_post_init
    self._init_metadata()
  File "C:\msys64\home\cd4mcq\packages\mx\mx.py", line 9164, in _init_metadata
    self._load_metadata()
  File "C:\msys64\home\cd4mcq\packages\mx\mx.py", line 9397, in _load_metadata
    self.mx_register_dynamic_suite_constituents(_register_project, _register_distribution)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 1622, in mx_register_dynamic_suite_constituents
    register_distribution(get_final_graalvm_distribution())
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 1480, in get_final_graalvm_distribution
    _final_graalvm_distribution = GraalVmLayoutDistribution('GraalVM', _base_graalvm_layout)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 438, in __init__
    if _force_bash_launchers(get_polyglot_launcher_project().native_image_config):
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 1561, in get_polyglot_launcher_project
    "destination": "polyglot",
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 924, in __init__
    super(GraalVmPolyglotLauncher, self).__init__(suite, GraalVmLauncher.launcher_project_name(launcher_config), deps, workingSets, launcher_config, **kw_args)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 870, in __init__
    super(GraalVmLauncher, self).__init__(suite, name, deps, workingSets, native_image_config, theLicense=theLicense, **kw_args)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 809, in __init__
    stage1 = get_stage1_graalvm_distribution()
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 1470, in get_stage1_graalvm_distribution
    _stage1_graalvm_distribution = GraalVmLayoutDistribution('GraalVM', _base_graalvm_layout, stage1=True)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 489, in __init__
    **kw_args)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 209, in __init__
    check_versions(join(_jdk_dir, _src_jdk_base), anyjdk_version_regex, graalvm_version_regex=graalvm_version_regex, expect_graalvm=False, check_jvmci=False)
  File "C:\msys64\home\cd4mcq\packages\graal\vm\mx.vm\mx_vm.py", line 1906, in check_versions
    mx.abort("'{}' has an unexpected version string:\n{}\ndoes not match:\n{}".format(jdk_dir, out, jdk_version_regex.pattern))
  File "C:\msys64\home\cd4mcq\packages\mx\mx.py", line 12420, in abort
    traceback.print_stack()
'C:\Program Files\AdoptOpenJDK\jdk-11.0.3.7-hotspot\.' has an unexpected version string:
openjdk version "11.0.3" 2019-04-16
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.3+7)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.3+7, mixed mode)
does not match:
(openjdk|java) version \"(?P<jvm_version>[0-9a-z_\-.]+)\".*\n(OpenJDK|Java\(TM\) SE) Runtime Environment [ 0-9.]*\(build [0-9a-z_\-.+]+\)
```

**Comment by gilles-duboscq** [comment]
Hi @oroppas, thank you for the report.
I'm pushing a fix through.

**Comment by gilles-duboscq** [comment]
This should be fixed by 68bb1e0d8.

**Comment by oroppas** [comment]
Thanks for the fix!
